app/src/main/java/org/fossasia/pslab/activity/OscilloscopeActivity.java

```diff
@@ -1075,22 +1096,38 @@ protected void onPostExecute(Void aVoid) {
                 isCH2FrequencyRequired = false;
             }
             if (isAstableMultivibratorExperiment) {
-                LineDataSet dataSet1;
+                LineDataSet dataset1;
                 LineDataSet dataSet2;
-                dataSet1 = new LineDataSet(entries1, "CH1 INPUT");
+                dataset1 = new LineDataSet(entries1, "CH1 INPUT");
                 dataSet2 = new LineDataSet(entries2, "CH2 OUTPUT");
-                dataSet1.setColor(Color.GREEN);
+                dataset1.setColor(Color.GREEN);
                 dataSet2.setColor(Color.RED);
-                dataSet1.setDrawCircles(false);
+                dataset1.setDrawCircles(false);
```

Comment:
I removed all typos in PR #557. This PR introduces them again.
Please resolve merge conflicts carefully.